### PR TITLE
Update mac_tool.py (#3.8)

### PR DESCRIPTION
**Pull request by KVS85**
### Scope & Purpose

Backport of https://github.com/arangodb/arangodb/pull/16434.

- [x] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

**Comment by KVS85** [comment]
- [ ] https://jenkins.arangodb.biz/job/arangodb-ANY-mac-matrix.x86-64-2/4/

**Comment by cw00dw0rd** [comment]
@cla-bot check

**Comment by KVS85** [comment]
- [ ] https://jenkins.arangodb.biz/job/arangodb-ANY-mac-matrix.x86-64-2/7/

**Review by dothebart — APPROVED** [review]

**Review by fceller — APPROVED** [review]
Lgtm

File: tests/Auth/UserManagerClusterTest.cpp [hunk 1]
```diff
@@ -32,7 +32,14 @@ namespace tests {
...
 
 namespace {
+#if defined(__APPLE__) && __clang_major__ == 13 && __clang_minor__ == 1
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-variable"
+#endif
 static char const* FailureOnLoadDB = "UserManager::performDBLookup";
+#if defined(__APPLE__) && __clang_major__ == 13 && __clang_minor__ == 1
+#pragma clang diagnostic pop
+#endif
```
Comment (MBkkt) [inline tests/Auth/UserManagerClusterTest.cpp:42]: It's unused only in non-maintainer mode
```suggestion
[[maybe_unused]] static char const* FailureOnLoadDB = "UserManager::performDBLookup";
```

**Review by MBkkt — COMMENTED** [review]
Suggest this https://github.com/arangodb/arangodb/pull/16445

**Comment by KVS85** [comment]
- [ ] https://jenkins.arangodb.biz/job/arangodb-ANY-mac-matrix.x86-64-2/13/

**Pull request merged**